### PR TITLE
Savings Container

### PR DESCRIPTION
**Pull request by laurenho025**

### CODE REVIEW

**Comment by laurenho025** [comment]
Screen Shot 2020-02-26 at 8 41 53 PM

**Comment by laurenho025** [comment]
Screen Shot 2020-03-02 at 10 48 46 PM

**Comment by laurenho025** [comment]
Screen Shot 2020-03-02 at 11 03 44 PM

File: App/Containers/ChartScreen/ChartScreen.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ export default class ChartScreen extends React.Component {
   render() {
     const labels = lastNDays(7);
     return (
-      <View style={[Helpers.fillRowCenter, styles.container]}>
-        <Chart title="Past savings" data={[-10, -15, 40, 19, 32, 15, 52]} xLabels={labels}/>
```
Comment (uawasth2) [inline App/Containers/ChartScreen/ChartScreen.js]: Why get rid of the title?

Comment (laurenho025) [reply App/Containers/ChartScreen/ChartScreen.js]: I think I took it out because its not in the mockup, but I just added it back in.

**Pull request merged**